### ISSUE TITLE
Contributing instructions don't seem to work at all.

### ISSUE THREAD

**Issue by Cyp** [babel/minify#836]
When reporting this, it says: `Also have a look at the Debugging guidelines: https://github.com/babel/minify/blob/master/CONTRIBUTING.md#debugging`, but there is no debugging section on the page.

**Describe the bug**

The instructions at `https://github.com/babel/minify/blob/master/CONTRIBUTING.md` don't actually work.

**To Reproduce**

Try following the instructions. First git clone babel/babel, and build it, in case that helps with anything.

Then try git cloning babel/minify, and following the instructions.

```
$ make bootstrap
make: *** No rule to make target 'bootstrap'.  Stop.
```

Something actually happens if running `npm run bootstrap && npm run build`, though.

Then try using it.

```js
'use strict';

require('@babel/core').transform('console.log("Hello world");', {presets: ['minify']});
```

Gives some error. Tried `npm install`. Then it instead gives:
`Error: Cannot find module 'babel-preset-minify' from '~/git/babel/minify'`

Then try
```js
'use strict';

require('@babel/core').transform('console.log("Hello world");', {plugins: ['minify-mangle-names']});
```
which gives
```
Error: Cannot find module 'babel-plugin-minify-mangle-names' from '~/git/babel/minify'`
$ ls packages/babel-plugin-minify-mangle-names
lib  package.json  README.md  src  __tests__
```
Then try
```js
'use strict';

require('@babel/core').transform('console.log("Hello world");', {plugins: ['./packages/babel-plugin-minify-mangle-names']});
```
which gives a different error `Error: Cannot find module 'babel-helper-mark-eval-scopes'`.

Then just give up, and report this as a bug.

**Expected Output**

```js
'use strict';console.log("Hello world");
```

**Possible solution**

No apparent solution. It seems hopeless.

**Comment by vigneshshanmugam** [comment]
Sorry, it wasn't clear at the first place. I have fixed the debugging section link in #838 

We have a dedicated [docs](https://github.com/babel/minify/tree/master/docs) folder that can help with setting up the project and running tests, benchmarks locally.

**Comment by vigneshshanmugam** [comment]
You would need to follow along the instructions for setting up the project locally here https://github.com/babel/minify/blob/master/docs/setup.md
